### ISSUE TITLE
Error parsing extension manifest when launching "normal" NW.js build in Linux

### ISSUE THREAD

**Issue by sysrage** [nwjs/nw.js#7575]
NWJS Version : 0.48.2
Operating System : Linux (has been reproduced in CentOS 8.2, Fedora 32, and Ubuntu 20.04.1)

### Expected behavior

When launching the "normal" (non-SDK) build of NW.js, an error should not be displayed regarding the parsing of an extension manifest.

### Actual behavior

When launching the "normal" (non-SDK) build of NW.js, the following error is displayed:

```
[802819:802819:1222/230818.000881:ERROR:component_loader.cc(183)] Failed to parse extension manifest.
```

This appears whether NW.js is being launched alone (no package.json or manifest) or with a full application. This does *not* appear if the SDK build of NW.js is used.

Please note that this appears to be the same as issue #7069, but that issue has had no traction for over a year.

### How to reproduce

1. Boot Linux
2. Download NW.js
3. Run NW.js (./nw)
4. Note the error message.

**Comment by Ravens0610** [comment]
Same thing happened

**Comment by rogerwang** [comment]
This is fixed in git and will be available in the next [nightly build](http://dl.nwjs.io/live-build/).

**Comment by wyeric1987** [comment]
Is this fixed? I'm getting this same error on Ubuntu 20.04 and Configurator 10.7.1

./betaflight-configurator 
[4531:4531:1224/235446.885716:ERROR:component_loader.cc(169)] Failed to parse extension manifest.

**Comment by sysrage** [comment]
> Is this fixed? I'm getting this same error on Ubuntu 20.04 and Configurator 10.7.1
> 
> ./betaflight-configurator [4531:4531:1224/235446.885716:ERROR:component_loader.cc(169)] Failed to parse extension manifest.

Yes, this is resolved and has been for several builds. I just tested with 0.59.1 and do not see this error message. If you're seeing this message with a recent build, open a new issue with full details.
